docs/topics/settings.rst

```diff
@@ -218,6 +218,26 @@ Default: ``None``
 
 The name of the region associated with the AWS client.
 
+.. setting:: ASYNCIO_EVENT_LOOP
+
+ASYNCIO_EVENT_LOOP
+------------------
+
+Default: ``None``
+
+Import path of a given asyncio event loop.
```

Comment:
```suggestion
Import path of a given asyncio event loop class.
```

:lipstick: It may be the most obvious interpretation, but better not to leave room for doubt.